### DISCUSSION TITLE
제페토 내에서 실제로 사용할 수 있는 아이템을 지급 할 수 있나요?

### DISCUSSION THREAD

**Discussion by sunten9497** [naverz/zepeto-studio-kor#30] [5. Q&A]
게임을 모두 클리어한 유저에게 최종적으로 보상 아이템을 줄 예정인데, 실제로 가능한지 , 
어떤 방식으로 해야 가능한지를 묻고싶습니다.

**Comment by mentalcrash** [comment]
안녕하세요.
질문이 제페토에서 사용하는 재화를 말씀하시는 건가요?
제페토 사용 재화(코인)을 보상으로 주는 기능은 내부 개발 중입니다.

**Reply by sunten9497** [reply]
최종적으로 맵을 클리어하면 ufo라는 탈 것을 유저한테 부여하고싶은데 그것이 다른 제패토 맵에서도 적용될수있게 하고싶습니다.

**Reply by hyeonjin-jo** [reply]
안녕하세요

제페토맵 단위로 아이템을 제작하시는 것은 가능하나  다른 맵과 공유되는 아이템은 현재 고려되고 있지 않습니다.
